admin/tests/unit/models/session-test.js

```diff
@@ -151,4 +133,251 @@ module('Unit | Model | session', function(hooks) {
+      const countNotCheckedEndScreen = sessionWithOneUncheckedEndScreen.countNotCheckedEndScreen;
+      assert.equal(countNotCheckedEndScreen, 1);
+    });
+
+    test('it should count 0 unchecked box if only one box (checked)', function(assert) {
+      const countNotCheckedEndScreen = sessionWithOneCheckedEndScreen.countNotCheckedEndScreen;
+      assert.equal(countNotCheckedEndScreen, 0);
+    });
+
+  });
+
+  module('#countNonValidatedCertifications', function() {
+
+    let sessionWithOneNotValidatedCertif;
+    let sessionWithValidatedCertif;
+
+    hooks.beforeEach(async function() {
+      sessionWithOneNotValidatedCertif = run(() => {
+        const certif = store.createRecord('certification', { status: 'nonValidated' });
+        return store.createRecord('session', { certifications: [certif] });
+      });
+
+      sessionWithValidatedCertif = run(() => {
+        const certif = store.createRecord('certification', { status: 'validated' });
+        return store.createRecord('session', { certifications: [certif] });
+      });
+    });
+
+    test('it should count 1 non validated', function(assert) {
+      const countNonValidated = sessionWithOneNotValidatedCertif.countNonValidatedCertifications;
+      assert.equal(countNonValidated, 1);
+    });
+
+    test('it should count 0 non validated', function(assert) {
+      const countNonValidated = sessionWithValidatedCertif.countNonValidatedCertifications;
+      assert.equal(countNonValidated, 0);
+    });
+
+  });
+
+  module('#countPublishedCertifications', function() {
+
+    let sessionWithOnePublishedCertif;
+    let sessionWithNoPublishedCertif;
+
+    hooks.beforeEach(async function() {
+      sessionWithOnePublishedCertif = run(() => {
+        const certif = store.createRecord('certification', { isPublished: true });
+        return store.createRecord('session', { certifications: [certif] });
+      });
+
+      sessionWithNoPublishedCertif = run(() => {
+        const certif = store.createRecord('certification', { isPublished: false });
+        return store.createRecord('session', { certifications: [certif] });
+      });
+    });
+
+    test('it should count 1 published', function(assert) {
+      const countPublished = sessionWithOnePublishedCertif.countPublishedCertifications;
+      assert.equal(countPublished, 1);
+    });
+
+    test('it should count 0 published', function(assert) {
+      const countNonValidated = sessionWithNoPublishedCertif.countPublishedCertifications;
+      assert.equal(countNonValidated, 0);
+    });
+
+  });
+
+  module('#displayDate', function() {
+
+    test('it should display date without time properly', function(assert) {
+      // given
+      const session = store.createRecord('session', { date: '2020-01-01' });
+
+      // when
+      const displayDate = session.displayDate;
+
+      // then
+      assert.equal(displayDate, new Date('2020-01-01').toLocaleString('fr-FR', { day: 'numeric', month: 'numeric', year: 'numeric' }));
```

Comment:
C'est dommage (et pas très parlant) de répéter l'implémentation dans le test.
Suggestion :

```suggestion
      assert.equal(displayDate, '01/01/2020');
```

(même suggestion pour les autres tests ci-dessous)